README.md

```diff
@@ -43,24 +55,29 @@ export PATH="$PWD/target/debug:$PATH"
 # Import the optional helper function `linera_spawn_and_read_wallet_variables`.
 source /dev/stdin <<<"$(linera net helper 2>/dev/null)"
 
-# Run a local test network with the default parameters and 0 extra user wallets.
-# This will set environment variables LINERA_{WALLET,STORAGE}_0 referenced by -w0 below.
-linera_spawn_and_read_wallet_variables linera net up --extra-wallets 0
+# Run a local test network with the default parameters and a number of microchains
+# owned by the default wallet.
+# The helper function will set the 2 environment variables LINERA_{WALLET,STORAGE}.
+linera_spawn_and_read_wallet_variables \
+linera net up
 
 # Print the set of validators.
-linera -w0 query-validators
+linera query-validators
 
-# Query some of the chains created at genesis
+# Query the default balance (aka the "chain balance") of some of the chains.
```

Comment:
Maybe just "chain balance"? ("Default" sounds like how much a new chain would have in the beginning or something.)